src/reducers/versions.tsx

```diff
@@ -509,6 +513,47 @@ export const getRelativeDiffAnchor = ({
   return null;
 };
 
+export type GetRelativeDiffParams = {
+  _findRelativePathWithDiff?: typeof findRelativePathWithDiff;
+  _getRelativeDiffAnchor?: typeof getRelativeDiffAnchor;
+  currentAnchor: string | void;
+  diff: DiffInfo;
+  pathList: string[];
+  position?: RelativePathPosition;
+  version: Version;
+};
+
+type RelativeDiffResult = {
+  anchor: string | null;
+  path: string | null;
+};
+
+export const getRelativeDiff = ({
+  _findRelativePathWithDiff = findRelativePathWithDiff,
+  _getRelativeDiffAnchor = getRelativeDiffAnchor,
+  currentAnchor,
+  diff,
+  pathList,
+  position = RelativePathPosition.next,
```

Comment:
The coverage report says that the default value for this is not executed by any tests. This makes me think that you don't need to set a default for `position` -- why not just make it required? I think it would be better if the calling code always explicitly set a position.